howso/client/base.py

```diff
@@ -4885,3 +4885,25 @@ def set_params(self, trainee_id: str, params: Mapping):
 
         self.execute(trainee_id, "set_internal_parameters", parameters)
         self._auto_persist_trainee(trainee_id)
+
+    def clear_imputed_data(self, trainee_id: str, session: str, *, impute_session: t.Optional[str]):
+        """
+        Clears values that were imputed during a specified session.
+
+        Won't clear values that were manually set by the user after the impute.
+
+        Parameters
+        ----------
+        trainee_id : str
+            The id of the trainee.
+        session : str
```

Comment:
Is this the session assigned to the edit? If so, should this instead just use the active session like other methods?